core/src/test/java/nl/nn/adapterframework/util/CounterStatisticTest.java

```diff
@@ -0,0 +1,67 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * CounterStatistic Tester.
+ *
+ * @author <Sina Sen>
+
+ */
+public class CounterStatisticTest {
+
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction01() throws Exception {
+        CounterStatistic cs = new CounterStatistic(10);
+        cs.performAction(0);
+        assertEquals(cs.getValue(), 10);
+    }
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction2() throws Exception {
+        CounterStatistic cs = new CounterStatistic(10);
+        cs.performAction(2);
+        assertEquals(cs.getValue(), 0);
+    }
+    
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction34() throws Exception {
```

Comment:
Try to use more descriptive names.  testAction1, testAction2, etc. is a bit vague when you're going through the log files :)

Reply:
Indeed, the naming I used looks terrible, I used numbers to reflect the action with number given in the CounterStatistic class, however using the action names rather than the number will be more descriptive. I am changing that.